packages/utils/telemetry-utils/src/index.ts

```diff
@@ -2,6 +2,15 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
+
+import { ITelemetryBaseLogger } from "@fluidframework/common-definitions";
+
+//* Where should this actually go? common-definitions?
+declare global {
+    /** This function may be provided by the environment, like a mocha test hook */
+    export const getTestLogger: (() => ITelemetryBaseLogger) | undefined;
```

Comment:
i'd prefer it this couldn't be undefined, basically it needs to exist. you could actually create a dummy in mocha-test-setup, which already overrides console and such, so i think it fits

Reply:
Nice, I wanted that too but didn't spend much time thinking about where/how.